src/components/common/Application.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState, type FormEvent } from "react";
+import axios from "axios";
+import { Alert, Autocomplete, Input, TextField } from "@mui/material";
+
+const skills = [
+  'Full-stack', 'Front-end', 'Back-end', 'Dev ops', 'UI/UX design', 'Prompt engineering', 'Rust', 'Python', 'C++', 'Golang', 'Graphic Design', 'Video Editing', 'Product Management', 'Accounting', 'Law', 'Sales', 'Business Operations', 'Others'
+];
+
+const books = [
+  'Mastering Bitcoin', 'Mastering Lightning Network', 'BPD', 'LPD', 'Learning Bitcoin through Command Line', 'Programming Bitcoin', 'The Bitcoin Standard', 'Sovereign Individual', 'The Broken Money', 'The Blocksize War', 'Others'
+];
+
+const Application = (
+  { cohortName, regOpen }: { cohortName: string, regOpen: boolean }
+) => {
+  const [formData, setFormData] = useState({
+    name: "",
+    email: "",
+    location: "",
+    describeYourself: "",
+    year: "",
+    background: "",
+    github: "",
+    time: "",
+    why: "",
+    skills: [],
+    books: [],
+    enrolled: false,
+    role: cohortName
+  });
+  const [isRegOpen, setIsRegOpen] = useState(regOpen);
```

Comment:
We dont need this state. We can directly use regOpen

Reply:
makes sense!